### PR TITLE
fix(RHTAPWATCH-532): Improve dashboard deployment docs

### PR DESCRIPTION
**Pull request by amisstea**

### CODE REVIEW

**Review by Kousalya1998 — APPROVED** [review]
/lgtm

**Comment by amisstea** [comment]
@johnmcollier does this help?

File: README.md [hunk 1]
```diff
@@ -65,6 +65,11 @@ https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/m
 to learn how to develop AppSRE dashboards for RHTAP. This repository serves as
 versioned storage for the [dashboard definitions](dashboards/) and nothing more.
 
+Dashboards are automatically deployed to [stage](https://grafana.stage.devshift.net) AppSRE Grafana when merged into the `main` branch.
+Deploying to [production](https://grafana.app-sre.devshift.net/) requires an update of a commit
```
Comment (yftacherzog) [inline README.md:69]: Do our metrics reach Grafana stage? We have different datasources for them in Grafana prod. Do we also have them sent to stage?

**Review by Omeramsc — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (amisstea) [reply README.md:69]: Yes, there's a `rhtap-observatorium-stage` and `rhtap-observatorium-production` datasource in both Grafana instances.

**Review by yftacherzog — APPROVED** [review]

**Review by johnmcollier — COMMENTED** [review]
Hey, sorry, missed this before the weekend. I see it's already merged, but giving my +1 here

**Pull request merged**